src/main/java/org.badcode/TicTacToe.java

```diff
@@ -0,0 +1,280 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+
+
+   O | O | O
+  -----------
+   O | O | O 
+  -----------
+   O | O | O
+
+
+ */
+
+import java.util.Scanner;
+
+public class TicTacToe {
+
+	/**
+	 * @param args the command line arguments
+	 * @throws java.lang.InterruptedException
+	 */
+	public static void main(String[] args) throws InterruptedException {
```

Comment:
Codacy found an issue: [The method 'main(String)' has an NPath complexity of 283025, current threshold is 200](https://app.codacy.com/gh/EddieDemoOrg/petstore/pullRequest?prid=6129301)